src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -28,9 +28,12 @@ class BlockchainSuite extends FreeSpec with Matchers with Logger {
     "bcTotalDifficultyTest/uncleBlockAtBlock3afterBlock4" -> Set.empty,
     "TransitionTests/bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain"  -> Set.empty,
     "TransitionTests/bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain2"  -> Set.empty,
-    "TransitionTests/bcFrontierToHomestead/HomesteadOverrideFrontier" -> Set.empty
+    "TransitionTests/bcFrontierToHomestead/HomesteadOverrideFrontier" -> Set.empty,
```

Comment:
Please add a comment explaining why DAO test these are being ignored. @AlanVerbner do you remember why they weren't working for older versions of the tests repository?

Reply:
@rtkaczyk I was only running `bcHomesteadToDao` [tests](https://github.com/ethereum/tests/tree/develop/BlockchainTests/TransitionTests/bcHomesteadToDao) and those weren't working because the name and the comments didn't match (it was saying the transition was at Block#5 but the doc was referring to Block#8)

Reply:
So maybe setting the configs to block 8 would work with the version we currently use? Anyway, we can just re-enable them after EIP-684.

Reply:
Setting config to block 8 did not help. We will re-enable this test after updating blockchain suite